docs/rgp-lua/finaleplugin-properties.md

```diff
@@ -342,20 +342,20 @@ finaleplugin.AdditionalPrefixes = [[
 
 If you use additional menu properties, you'll often want to categorize your script as a group script. That way, users trying to find and download your script on this website will see a title and description that represents the group of menu items, not just one of the individual menu items.
 
-#### ScriptGroupName (string)
+#### GroupScriptName (string)
```

Comment:
Please revert these changes to the docs. The original field names are correct. Here's where they're defined in the metadata parser.

https://github.com/finale-lua/lua-scripts/blob/6287620c7202978e44f944a83da2150e956a6503/.github/actions/get-metadata/src/parse.ts#L172-L175

This may have accidentally been included since admittedly I wrote the documentation wrong at first and fixed them in https://github.com/finale-lua/lua-scripts/pull/297 earlier today.